src/rkopenmdao/runge_kutta_scheme.py

```diff
@@ -102,7 +105,7 @@ def compute_step(
             delta_t * self.butcher_tableau.butcher_weight_vector,
             ((0,), (0,)),
         )
-        if self.butcher_tableau.is_embedded:
+        if self.butcher_tableau.is_embedded and self.use_adaptive_time_stepping:
```

Comment:
The case where the tableau has no embedded scheme, but the user requests adaptive time stepping, should at least give a warning, or even throw an error.